agent/util-scripts/pbench-tool-meister-start

```diff
@@ -340,6 +342,18 @@ def main(argv):
         ), f"bad channel: {resp!r}"
         assert resp["data"] == 1, f"bad data: {resp!r}"
 
+    # 2.5. Add tool metadata json to redis
+    try:
+        tm_start_path = Path(
+            Path(__file__).resolve(), "..", ".."
+        ).resolve(strict=True)
+    except FileNotFoundError:
+        raise Exception(f"missing {tm_start_path}")
+    except Exception:
+        raise
```

Comment:
Instead of raising an exception, let's consider using logger.exception with a suitable message.